### PR TITLE
feat(notched-outline): update to v1.1.1

### PR DESCRIPTION
**Pull request by moog16**
related #697 

NOTE: Text field and select goldens are not ok in this PR. I've checked in new "broken" goldens until they are fixed in their own subsequent PRs.

### CODE REVIEW

**Comment by gugu** [comment]
Sorry for the disappearing, there was a crazy week for a project

**Comment by moog16** [comment]
No worries :) I appreciate all the work you've already done. I'm going to push this release back a week or two so that we can get all of mdc web v1.x. How does that sound?

File: test/screenshot/select/index.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import Select, {SelectProps} from '../../../packages/select/index';
-import '../../../packages/select/index.scss';
-import './index.scss';
+// import '../../../packages/select/index.scss';
```
Comment (abhiomkar) [inline test/screenshot/select/index.tsx]: Remove commented code.

File: test/screenshot/text-field/index.tsx [hunk 2]
```diff
@@ -1,10 +1,7 @@
 import * as React from 'react';
 import {Link} from 'react-router-dom';
 import textFieldVariants from './variants';
-import '../../../packages/floating-label/index.scss';
-import '../../../packages/line-ripple/index.scss';
-import '../../../packages/notched-outline/index.scss';
-import '../../../packages/text-field/index.scss';
+// import '../../../packages/text-field/index.scss';
```
Comment (abhiomkar) [inline test/screenshot/text-field/index.tsx]: Same here.

**Pull request merged**